samples/drivers/spi_flash/sample.yaml

```diff
@@ -16,3 +16,10 @@ tests:
             - "Data read 55 aa"
             - "Data read matches with data written. Good!!"
     depends_on: spi
+  sample.driver.spi_flash_dpd:
+    tags: spi flash
+    filter: DT_INST_0_JEDEC_SPI_NOR_LABEL
```

Comment:
This `filter:` needs updating, we removed support for `DT_` symbol based filters, need to use a function like `filter: dt_compat_enabled("jedec,spi-nor")`